tokens/src/core/components/Dropdown.json

```
@@ -37,7 +37,13 @@
           "base": { "source": "$dropdown-padding-y", "$value": ".5rem" },
           "item": { "source": "$dropdown-item-padding-y", "$value": ".25rem" }
         },
-        "header": { "source": "$dropdown-header-padding", "$value": "{spacing.dropdown.padding.y.base} {spacing.dropdown.padding.x.item}" }
+        "header": {
+          "source": "$dropdown-header-padding",
+          "$value": {
+            "x": "{spacing.dropdown.padding.x.item}",
+            "y": "{spacing.dropdown.padding.y.base}"
```

After playing around with this on the docs site a bit it seems like having the y padding tied to the menu base y padding is a bit confusing. I wouldn't expect the padding above the header to jump from `1rem` when `dropdown-padding-y` is set to `0.5rem` to `2rem` when setting `dropdown-padding-y` to `1rem`.

To avoid confusion I think the best path forward here is to just have the `y` value for header padding defined as an independent value.

```suggestion
            "y": ".5rem"
```

---

Thanks, removed in [`0905115` (#3222)](https://github.com/openedx/paragon/pull/3222/commits/09051155b1aa93b654844015ef9289b0387736d2#diff-ad58aa35fe3f66fc8148f05a3e6dfd4bf47b9a943f7bb407521daedd23d6c573L44)